readme.md

```diff
@@ -0,0 +1,14 @@
+How to reproduce/execute the script in docker
+
+1. clone the project from the repo
+
+2. Build and run the Docker container
+    
+    docker build -t gpa-script:latest .
```

Comment:
Why not use `gpa-app` instead of `gpa-script`?

Reply:
fixed